test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -313,7 +318,11 @@ func (HeaderOperations) ResponseDateHandleResponse(resp *azcore.Response) (*Head
 	if !resp.HasStatusCode(http.StatusOK) {
 		return nil, newError(resp)
 	}
-	return &HeaderResponseDateResponse{StatusCode: resp.StatusCode}, nil
+	val, err := time.Parse(time.RFC3339, resp.Header.Get("value"))
```

Comment:
This isn't working properly yet. The issue is tracked in #270. I'm taking a look at it.